packages/modules-sdk/src/definitions.ts

```diff
@@ -15,6 +15,7 @@ export enum Modules {
   PRICING = "pricingService",
   PROMOTION = "promotion",
   AUTHENTICATION = "authentication",
+  WORKFLOW = "workflows",
```

Comment:
nit:
```suggestion
  WORKFLOW_ENGINE = "workflow-engine",
```

Change should be reflected in the rest of the file